src/profile_wizard.cpp

```diff
@@ -914,13 +952,20 @@ struct AutoConnectCamera
     GuideCamera *operator->() const { return m_camera; }
 };
 
-static void SetBinningLevel(wxWindow *parent, const wxString& selection, int val)
+static void SetBinningLevel(ProfileWizard *parent, const wxString& selection, int val)
 {
-    AutoConnectCamera cam(parent, selection);
+    AutoConnectCamera cam(parent, selection, false);
     if (cam && cam->MaxBinning > 1)
         cam->SetBinning(val);
 }
 
+static wxString CameraSelectionKey(const wxString& camName)
+{
+    std::hash<std::string> hash_fn;
+    std::string name(camName.c_str());
+    return wxString::Format("/cam_hash/%lx/whichCamera", (unsigned long) hash_fn(name));
+}
+

```

Comment:
Let's not have two copies of this code.  On way to fix it would be to move the original code into a public static method of GearDialog and call it from the places it is needed.